source/envs/env_starcraft.py

```
@@ -0,0 +1,500 @@
+                self.y + 30,
+                self.x + 1100,
+                self.y + 600,
+            )
+            unit_info = self.screen.screenshot_buffer[-1][750:, 280:810]
+            gray = cv2.cvtColor(unit_info, cv2.COLOR_BGR2GRAY)
+            _, thresh = cv2.threshold(gray, 100, 255, cv2.THRESH_BINARY)
+            text = pytesseract.image_to_string(thresh, lang="eng")
+
+            if "Artanis" in text or "Lv.2" in text:
+                return True
+            return False
```

Ruff (0.12.2)

Return the condition `bool("Artanis" in text or "Lv.2" in text)` directly

Replace with `return bool("Artanis" in text or "Lv.2" in text)`

(SIM103)